### PR TITLE
Fixed error on !user when specifying an offline user

### PR DESCRIPTION
**Pull request by minisbett**

### CODE REVIEW

File: app/commands.py [hunk 1]
```diff
@@ -898,14 +898,20 @@ async def user(ctx: Context) -> Optional[str]:
     else:
         last_np = None
 
+    osu_version = (
+        p.client_details.osu_version.date
+        if hasattr(p.client_details, "osu_version")
```
Comment (7ez) [inline app/commands.py:903]: do `if p.client_details is not None`

**Review by 7ez — CHANGES_REQUESTED** [review]
yes

File: app/commands.py (hunk 1 above)
Comment (cmyui) [reply app/commands.py:903]: `if p.online` should also suffice

Comment (cmyui) [reply app/commands.py:903]:
```suggestion
        if p.online
```

File: app/commands.py [hunk 2]
```diff
@@ -898,14 +898,20 @@ async def user(ctx: Context) -> Optional[str]:
...
+        else "Unknown"
+    )
     return "\n".join(
         (
             f'[{"Bot" if p.bot_client else "Player"}] {p.full_name} ({p.id})',
             f"Privileges: {priv_list}",
+            f"Donator status: {donator_status}",
```
[the remark this answers is not in the document]
Comment (cmyui) [reply app/commands.py:906]: will just remove for now so we can merge. feel free to make another pr
```suggestion
```

File: app/commands.py [hunk 3]
```diff
@@ -898,14 +898,15 @@ async def user(ctx: Context) -> Optional[str]:
     else:
         last_np = None
 
+    osu_version = p.client_details.osu_version.date if p.online else "Unknown"
```
Comment (cmyui) [inline app/commands.py:901]:
```suggestion
    osu_version = p.client_details.osu_version.date if p.online else "Unknown"

```

**Review by cmyui — APPROVED** [review]

**Comment by cmyui** [comment]
thank ya!

File: app/commands.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (minisbett) [reply app/commands.py]: oh yeah lmao that's from my code I just copy pasted

**Pull request merged**